File: ExampleScripts/PHP/playback.php
<?php

	/*
		Arguments passed:  <action:start/stop> %username% %device.name% %item.id% %item.name%
	*/

	$log_file = "d:\\embyscripts\\playback_php.log";

	$status = $argv[1];
	$username = $argv[2];
	$devicename = $argv[3];
	$item_id = $argv[4];
	$item_name = $argv[5];

	file_put_contents($log_file, "Playback: $status : User $username on device $devicename - media: ($item_id) $item_name", FILE_APPEND);
	

File: ExampleScripts/Python/playback.py
#!/usr/bin/python

# START/STOP "%item.name%" "%item.path%" "%item.type%" "%username%" "%device.name%"

import sys, getopt

playback_log_file = "c:\emby\scripts\playback_python.log"

if __name__ == "__main__":

    if (len(sys.argv) < 7):
       print(f"Usage: playback.py <playback_action> <playback_title> <playback_path> <media_type> <username> <device_name>")
       sys.exit()

    playback_action = sys.argv[1];
    media_title = sys.argv[2];
    media_path = sys.argv[3];
    media_type = sys.argv[4];
    username = sys.argv[5];
    device_name = sys.argv[6];

    f = open(playback_log_file, 'a')
    
    f.write ('User ' + username + ' Playback on Device ' + playback_action + ' for media (' + media_type + ') ' + media_title + ' (Path: ' + media_path + ')\n')


